Counterexamples/AharoniKorman.lean

```diff
@@ -71,6 +71,11 @@ aim of reaching a contradiction (as then, no such partition can exist). We may f
   we have a contradiction (`no_spinalMap`), and therefore show that no spinal map exists.
 -/
 
+#adaptation_note 2025-03-12
+/--
+We temporarily comment out this file on `nightly-testing`, awaiting fixes for a complex
+interaction between `omega`, `aesop`, and asynchronous elaboration.
+
```

Comment:
Is the idea that it's better to merge now and sort this out later?

Reply:
Yes, Joachim will fix this but he's travelling at the moment. We need to keep `nightly-testing` compiling every day or we lose the ability to test Lean PRs against Mathlib.